blocks/arrow-icon/arrow-icon.css

```diff
@@ -8,3 +8,11 @@
 .arrow-icon:hover {
   background: url("./yellow-arrow.png") center center no-repeat;
 }
+
+@media only screen and  (max-width: 320px) {
+  .arrow-icon {
+    font-size: 14px;
```

Comment:
The icon of the arrow is aligned to the upper row of the link and not the lower row

Reply:
updated